sql/PlayerBot/mangos_botguy_001.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO `gossip_menu_option` VALUES('0','16','0','GOSSIP_OPTION_BOT','99','1','0','0','0','0','0',NULL,'0','0','0','0','0','0','0','0','0');
```

Comment:
same here so there will be no need to have sql/playerbot folder. It will be part of the core and DB will be directly ready.
Note: in that particular case check if core part is required or not by adding that sql.